docs/platform/howto/byoc/enable-byoc.rst

```diff
@@ -0,0 +1,75 @@
+Enable bring your own cloud (BYOC)
+==================================
+
+Enabling :doc:`the bring your own cloud (BYOC) feature </docs/platform/concepts/byoc>` allows you to :doc:`create custom clouds </docs/platform/howto/byoc/create-custom-cloud>` in your Aiven organization.
+
+.. note::
+
+   Enabling :doc:`the BYOC feature </docs/platform/concepts/byoc>` or creating custom clouds in your Aiven environment does not affect your existing organizations, projects, or services. This only makes the new BYOC capabilities available in your environment.
```

Comment:
Should we be more specific here?  Enabling the BYOC features allows you to run Aiven services in your cloud provider account.